source/extensions/common/wasm/context.h

```diff
@@ -473,6 +474,10 @@ class Context : public proxy_wasm::ContextBase,
   // Filter state prototype declaration.
   absl::flat_hash_map<std::string, Filters::Common::Expr::CelStatePrototypeConstPtr>
       state_prototypes_;
+
+  // Listener specific information - set only for stream contexts.
```

Comment:
Regarding the comment, does network filter allow to use it as well? in Non-wasm it is allowed. And you do set  direction_ in network filter config 
I could be wrong since I don't have too much contexts on wasm :)

Reply:
yeas it does. In Wasm, "stream context"  means each Http or Tcp/Network stream, fyi:)